diff -ruN eccodes-python-2.45.0/setup.py eccodes-python-2.45.0-patch/setup.py
--- eccodes-python-2.45.0/setup.py	2026-01-15 17:08:53.000000000 +0100
+++ eccodes-python-2.45.0-patch/setup.py	2026-01-21 10:30:32.981819412 +0100
@@ -97,7 +97,7 @@
 # only add eccodeslib as dependency if building a distribution that should depend on it
 if binary:
     install_requires += [
-        "eccodeslib;platform_system!='Windows'",
+        "eccodes;platform_system!='Windows'",
     ]
 
 setuptools.setup(
